test/e2e/clients.go

```diff
@@ -8,8 +8,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 
-	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
-	nmoapi "kubevirt.io/node-maintenance-operator/pkg/apis"
+	//operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	nmoapi "kubevirt.io/node-maintenance-operator/api/v1beta1"
```

Comment:
just to make sure - the move to beta is intentional?

Reply:
maybe this is the new generated paty

Reply:
yes, it's just a changed package, API version was v1beta1 before as well